<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>894</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="vtkContainer" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>911</width>
      <height>551</height>
     </rect>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>901</width>
       <height>541</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_vtk"/>
    </widget>
   </widget>
   <widget class="QWidget" name="vtkContainerX" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>291</width>
      <height>261</height>
     </rect>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>291</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_vtk_5"/>
    </widget>
   </widget>
   <widget class="QWidget" name="vtkContainerY" native="true">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>570</y>
      <width>291</width>
      <height>261</height>
     </rect>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>291</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_vtk_6"/>
    </widget>
   </widget>
   <widget class="QWidget" name="vtkContainerZ" native="true">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>570</y>
      <width>301</width>
      <height>261</height>
     </rect>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>301</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_vtk_7"/>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>954</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
